src/bootstrap/configure.py

```diff
@@ -417,6 +417,8 @@ def parse_example_config(known_args, config):
         # Avoid using quotes unless it's necessary.
         targets[target][0] = targets[target][0].replace("x86_64-unknown-linux-gnu", "'{}'".format(target) if "." in target else target)
 
+    if 'profile' not in config:
+        config['profile'] = 'user'
```

Comment:
I think we should inform the dev with a simple log

Reply:
How does this look?
```
; ./configure 
configure: processing command line
configure: 
configure: build.configure-args := []
configure: profile              := user
configure: 
configure: writing `config.toml` in current directory
configure: 
configure: run `python /home/jyn/src/rust3/x.py --help`
```

Reply:
this helps good enough, thanks